ui.apps/src/main/content/jcr_root/apps/etoolbox-rollout-manager/clientlibs/rollout-manager-ui/js/console-ui.dialog.js

```diff
@@ -59,11 +64,37 @@
         dialog.footer.appendChild(closeBtn);
     }
 
-    function insertLogItem(dialog, message, safe) {
-        const logItem = document.createElement('div');
-        logItem.className = 'rollout-manager-log-item';
-        logItem[safe ? 'textContent' : 'innerHTML'] = message;
-        dialog.content.insertAdjacentElement('beforeend', logItem);
+    function updateLog(dialog, message) {
+        if (message.type !== 'rollout' && message.type !== 'activation') return;
+        const rolloutItems = $(dialog).find('.rollout-log-item');
+        const itemToUpdate = rolloutItems.filter((i, item) => item.value === message.path);
```

Comment:
A little better and more readable. And also a guard for the absence of an itemToUpdate.
```suggestion
    const itemToUpdate = $(dialog)
        .find('.rollout-log-item')
        .filter((i, item) => item.value === path)
        .first();
    if (!itemToUpdate.length) return;
```